packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -32,7 +33,6 @@
   "governanceTokenSymbol": "OP",
   "governanceTokenOwner": "0x5C4e7Ba1E219E47948e6e3F55019A647bA501005",
   "l2GenesisBlockGasLimit": "0x1c9c380",
-  "l2GenesisBlockCoinbase": "0x4200000000000000000000000000000000000011",
```

Comment:
Why'd this get removed?

Reply:
It is always hardcoded to that value, its legacy from when it was configurable